src/main/java/com/stripe/model/Invoice.java

```diff
@@ -244,11 +244,6 @@
   @SerializedName("paid")
   Boolean paid;
 
-  /**
-   * The PaymentIntent associated with this invoice. The PaymentIntent is generated when the invoice
-   * is finalized, and can then be used to pay the invoice. Note that voiding an invoice will cancel
-   * the PaymentIntent.
-   */
```

Comment:
we should keep the docs here. I think we internally hid this for a bit but still

Reply:
Huh i realized it's a problem in OpenAPI generator here. 
Will make a fix.

Reply:
Okay since it's an old problem and I have a working PR. We can punt this first.